python/py-MDAnalysis/Portfile

```diff
@@ -25,28 +25,26 @@ master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  6ac6c5ad4ad83ff1227ec596cf47de76587d4478 \
-                    sha256  f45a024aca45e390ff1c45ca90beb2180b78881be377e2a1aa9cd6c109bcfa81 \
-                    size    19552874
-
-patchfiles          patch-unistd.diff
+checksums           rmd160  ece8d0a6ab8b0bb8e5625d547ebd821ec0c1ff20 \
+                    sha256  aa3079d1a82305eba58cf567fac8fc231940184ed88f9a4451be8433f4a06d3e \
+                    size    3436654
 
 if {${name} ne ${subport}} {
     # OpenMP is disabled now
     # See https://github.com/macports/macports-ports/pull/3902 for an attempt to fix the issue
     build.env-append        MDA_USE_OPENMP=0
     depends_build-append    port:py${python.version}-setuptools
     depends_lib-append      port:py${python.version}-biopython \
-                            port:py${python.version}-GridDataFormats \
-                            port:py${python.version}-gsd \
                             port:py${python.version}-joblib \
                             port:py${python.version}-matplotlib \
                             port:py${python.version}-mock \
-                            port:py${python.version}-mmtf-python \
                             port:py${python.version}-networkx \
                             port:py${python.version}-numpy \
                             port:py${python.version}-scipy \
                             port:py${python.version}-six \
                             port:py${python.version}-tqdm
+                            port:py${python.version}-mmtf-python \
+                            port:py${python.version}-GridDataFormats \
+                            port:py${python.version}-gsd \
```

Comment:
there should be no line continuation for the last dependency here (and yes, please keep them alphabetically sorted)

Reply:
In https://github.com/macports/macports-ports/pull/12970/commits/61130190b0c974a6b9975bd0c6cddacb75f951fa I corrected depends_lib order